comment/utils.py

```diff
@@ -75,7 +75,7 @@ def has_valid_profile():
 
 
 def is_comment_admin(user):
-    if settings.COMMENT_FLAGS_ALLOWED:
+    if settings.COMMENT_FLAGS_ALLOWED or settings.COMMENT_ALLOW_BLOCKING_USERS:
```

Comment:
may be we can move this to something along the lines of `_is_moderation_enabled`. it could be used in both `is_comment_admin` and `is_comment_moderator`.